File: AndroidBasics/notes/src/withoutGms/java/pan/alexander/notes/data/database/AppDatabase.java
package pan.alexander.notes.data.database;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import pan.alexander.notes.domain.entities.Note;
import pan.alexander.notes.domain.entities.Trash;

@Database(entities = {Note.class, Trash.class}, version = 2)
public abstract class AppDatabase extends RoomDatabase {
    public abstract NotesDao getNotesDao();

    public abstract TrashDao getTrashDao();
}

File: TestWeatherApp/app/src/main/java/pan/alexander/testweatherapp/framework/database/AppDatabase.kt
package pan.alexander.testweatherapp.framework.database

import androidx.room.Database
import androidx.room.RoomDatabase
import androidx.room.TypeConverters
import pan.alexander.testweatherapp.domain.entities.CurrentWeather

@Database(entities = [CurrentWeather::class], version = 1)
@TypeConverters(DatabaseConverters::class)
abstract class AppDatabase : RoomDatabase() {
    abstract fun currentWeatherDao(): CurrentWeatherDao
}
